src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,119 @@
+package duke;
+import duke.exception.*;
+
+import java.util.Scanner;
+
+public class Parser {
+    public static final int TODO_LENGTH = 4;
+    public static final int EVENT_LENGTH = 5;
+    public static final int DEADLINE_LENGTH = 8;
+
+    public static Scanner in = new Scanner(System.in);
+    public static String input;
+    public static TaskManager taskList = new TaskManager();
+
+    public static String getInput() {
+        String input = in.nextLine();
+        return input;
+    }
+
+    //Classify the input to a specific command
+    public static boolean isList() {
+        return input.equals("list");
+    }
+
+    public static boolean isBye() {
+        return input.equals("bye");
+    }
+
+    public static boolean isDone() {
+        return input.startsWith("done");
+    }
+
+    public static boolean isEvent() {
+        return input.startsWith("event");
+    }
+
+    public static boolean isDeadLine() {
+        return input.startsWith("deadline");
+    }
+
+    public static boolean isToDo() {
+        return input.startsWith("todo");
+    }
+
+    public static boolean isDelete() { return input.startsWith("delete"); }
+
+    //check if there is input exception
```

Comment:
As mentioned earlier, please maintain spacing and add comments which properly describe the condition.